ls1intum/artemis-android
# `Improvement`:  Small adjustment to  Code of Conduct sheet

### Problem Description
Changed the Code-of-Conduct bottom sheet from a half-screen modal to a full-screen display. The partial expansion previously blocked scrollable content, so adopting a full‐screen approach ensures all information is accessible in one click.

Note: This is a relatively small PR aimed to aligning the behavior with iOS and improving usability.
	
### Changes
 
• Updated the Code-of-Conduct bottom sheet to always open as a full-screen modal on Android.
• Added slight padding for improved layout consistency, particularly for first-time acceptance of the Code of Conduct.

### Steps for testing
1.	Open the Code of Conduct bottom sheet on both Android and iOS.
	•	Confirm that the modal now matches iOS behavior, opening as a full-screen modal.
2.	Test first-time acceptance:
	•	Verify that there is a small but noticeable gap (padding) between the text and header menu for better visual alignment.

### Screenshots

<img width="150" alt="Screenshot 2024-12-22 at 22 25 54" src="https://github.com/user-attachments/assets/abb8ba72-03a2-4c04-b03f-ba2d212a02ec" />
<img width="150" alt="Screenshot 2024-12-22 at 22 25 41" src="https://github.com/user-attachments/assets/841867b1-d8bf-4e8b-a021-9ef57bca8185" />
